record_rgb_depth/Makefile

```diff
@@ -0,0 +1,3 @@
+LDFLAGS = $(shell pkg-config opencv --libs) -lboost_filesystem
```

Comment:
Are you using the boost provided by ILLIXR?

Reply:
ILLIXR installs all libraries of boost, including filesystem. I have never manually installed boost standalone, so I guess that one is the boost provided by ILLIXR.